packages/string/__tests__/outdent.test.ts

```
@@ -0,0 +1,228 @@
+        `;
+        expect(result).toBe("    Some text");
+
+        const result2 = outdent`
+                ${outdent}
+            12345678
+        `;
+        expect(result2).toBe("5678");
+    });
+
+    it("should not get indentation from outdent when preceded by non-whitespace", () => {
+        const toString = "" + outdent;
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)